src/components/MetaData/index.test.tsx

```diff
@@ -18,1214 +18,1235 @@ jest.mock('react-dom/server', () => ({
 //   dangerouslyInjectJS: [],
```

Comment:
All of the spacing on this seems a bit jumbled. Is there a way of standardizing only places you've changed? Otherwise this looks good. I'm just concerned about regressions since I'm having a hard time reading this

Reply:
I wrote the tests from the ground up, there was a lot of cases that were not checking and a lot of missing test coverage. Now this component is on 97% test coverage (if remember well) and all test cases covered.
Yes the diff is complex in this case, was not only adding a couple tests case and fix others.